Documentation/contributing/development/dev_setup.rst

```diff
@@ -450,7 +450,9 @@ Minor version
 
 #. Run ``git add vendor/ test/provision/manifest/ Documentation/ && git commit -sam "Update k8s tests and libraries to v1.28.0-rc.0"``
 
-#. Submit all your changes into a new PR.
+#. Submit all your changes into a new PR. Ensure the PR is opened against a
+   branch ``cilium/cilium`` and *not* a fork. Otherwise, CI is not triggered
```

Comment:
```suggestion
   branch in ``cilium/cilium`` and *not* a fork. Otherwise, CI is not triggered
```

Reply:
Could be nice to mention where to ask for help if the developer does not have permission to create a branch in cilium/cilium?

Reply:
Thanks, good point. Fixed